modules/curse/curse.gd

```diff
@@ -1,45 +1,58 @@
 extends Node2D
 
 #const var
-const target_angle_epsilon = 0.1* TAU
-const line_circle_radius = 30 #pixel
-const pointerSpeed = 10 #pixel/second
-const pointerOffset = Vector2(0,-30) #pixel
-const skullSpeed = 10 #pixel/second
+const TARGET_ANGLE_EPSILON = 0.1* TAU
+const SKULL_SPEED = 10 #pixel/second
 
+const ANIM_IDLE = "Skull_Idle"
+const ANIM_INTRO = "skull_intro"
+const ANIM_OUTRO = "skull_outro"
 # editor var
 @export_node_path("Node2D") var INITIALLY_CURSED_CHARACTER : NodePath
 
 #ingame var
 var _cursed_character : Node2D = null
```

Comment:
I feel like these variables should be accessible from outside this script (its nice to be able to be able to know who is the cursed character). If so, one should rename _cursed_character to cursed_character.
I do not think it it necessary but one can use the set and get syntax to control accessibility:

var cursed_character : Node2D:
    set(value):
        # stuff
    get:
        # return stuff

Reply:
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html

Search for: Properties (setters and getters)